async.go

```diff
@@ -63,25 +63,45 @@ func (sr *snowflakeRestful) getAsync(
 	defer close(errChannel)
 	token, _, _ := sr.TokenAccessor.GetTokens()
 	headers[headerAuthorizationKey] = fmt.Sprintf(headerSnowflakeToken, token)
-	resp, err := sr.FuncGet(ctx, sr, URL, headers, timeout)
-	if err != nil {
-		logger.WithContext(ctx).Errorf("failed to get response. err: %v", err)
-		sfError.Message = err.Error()
-		errChannel <- sfError
-		return err
-	}
-	if resp.Body != nil {
+
+	var err error
+	var respd execResponse
+	retry := 0
+	retryPattern := []int32{1, 1, 2, 3, 4, 8, 10}
```

Comment:
why 10 and not 16?

Reply:
It's up 5 seconds. Why would it be 16?

`sleepTime := time.Millisecond * time.Duration(500*retryPattern[retry])`

Reply:
16 is in 2^4. So it's exponential back off.

Reply:
Oh but this pattern is 500ms * 10 so 5 seconds